src/components/Row.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+const Row = ({ guess, currentGuess }) => {
+  if (guess) {
+    return (
+      <div className="rows past">
+        {guess.map((selectedFact, idx) => (
+          <div key={idx} className={selectedFact.color}>
+            {selectedFact.key}
+          </div>
+        ))}
+      </div>
+    );
+  }
+
+  if (currentGuess) {
+    let facts = currentGuess.split(",");
+
+    return (
+      <div className="rows current">
+        {facts.map((fact, idx) => (
+          <div key={idx} className="filled">
+            {fact}
+          </div>
+        ))}
+        {[...Array(5 - facts.length)].map((_, idx) => (
+          <div key={idx}></div>
+        ))}
+      </div>
+    );
+  }
+
+  return (
+    <div className="rows">
+      <div></div>
```

Comment:
These should probably get a class name so that they're both more descriptive and more extensible in the future

Reply:
Aha... I thought it will be more dry since I didn't have any major use for them